docs/technology_stack.adoc

```diff
@@ -7,35 +7,42 @@
 toc::[]
 
 == Overview
-The following sections describe the technology stack used by Planner.
+The Planner is an Angular app built on top of Node.js. It can run as a standalone app, or be used as a library to be integrated into the link:./fabric8-platform[Fabric8 Platform].
 
-== Angular 4
-<TBD>
+Development or deployment of Planner is fairly platform agnostic, so we can skip the operating system requirements and discuss from the runtime environment to way-up.
 
-== CSS and SASS
-<WIP, to be changed to PF CSS, Less>
+== Development Runtime
+Planner uses *`Node.js`* as the platform-runtime, on top of which it builds; on that line, we use *`npm`* for resolving and distributing packages for the project.
```

Comment:
Imho, 'on top of which it builds;' 'on that line' these parts are not very clear.
Can we reword it to make it simpler? How about, 'Planner uses *`Node.js`* for building the platform-runtime and uses *`npm`* for resolving and distributing (can we mention here which packages?)packages for the project.' wdyt?
As per convention use bold for software packages *npm*